CONTRIBUTING.md

```diff
@@ -19,8 +19,9 @@ our own CI/CD: the ["superchain" image][superchain] from.
 The image can be built for local usage, too:
 
 ```console
+$ cd jsii # go to the root of the jsii repo
 $ IMAGE=superchain
-$ docker build -t ${IMAGE} ./superchain
+$ docker buildx build -t $IMAGE -f superchain/Dockerfile .
```

Comment:
Can we keep the old instructions here? Assuming they still work?
Open to offering both as alternative options though...

Reply:
I agree with the idea, but old instruction isn't working.

Because path of some `COPY` instruction now no longer match ([ref](https://github.com/aws/jsii/issues/4080#issue-1696159840)).

Reply:
No that's fine then! Thanks for letting me know!

Reply:
Thanks for your response, but I just want to make sure I understand correctly.

Does that mean I can leave the code as it is now?

Reply:
Yes, not request from me right now. I'll look into this section of doc myself but this will have to be my tomorrow!